h/security/policy/_cookie.py

```diff
@@ -41,7 +51,9 @@ def remember(self, request, userid, **kw):  # pylint:disable=unused-argument
             request.session.update(data)
             request.session.new_csrf_token()
 
-        return request.find_service(AuthCookieService).create_cookie(userid)
+        ticket_id = base64.urlsafe_b64encode(urandom(32)).rstrip(b"=").decode("ascii")
+        request.find_service(AuthTicketService).add_ticket(userid, ticket_id)
+        return self.cookie.get_headers([userid, ticket_id])
```

Comment:
It took a couple of passes to get this from SignedCookieProfile design,

you instantiate the cookie with no value, there no `set_value` but you rather have to `get_headers(value)` to set it (and get the haders).

You however do get_value to get the value.